cypress/integration/other/polymorphic-prop.spec.js

```diff
@@ -27,7 +27,7 @@ describe('Polymorphic prop', () => {
 
     it('overrides layout styles', () => {
       cy.get('#header')
-        .should('have.css', 'display', 'flex')
+        .should('have.css', 'display', 'block')
```

Comment:
Note that the actual expected value is "block", since it's the value on the custom styled header in that test scenario. Test suite was invalid previously.